### ISSUE TITLE
go1.6b1

### ISSUE THREAD

**Issue by cryptix** [ipfs/go-ipfs#2108]
In Go [1.6b1](https://tip.golang.org/doc/go1.6#go_command)  and later, you no longer need to use the `GO15VENDOREXPERIMENT` env var.  On the flipside you now see this when compiling ipfs:

```
package github.com/ipfs/go-ipfs/repo/fsrepo
        imports github.com/ipfs/go-ipfs/vendor/QmQg1J6vikuXF9oDvm4wpdeAUvvkVEKW1EYDw9HhTMnP2b/go-log: must be imported as QmQg1J6vikuXF9oDvm4wpdeAUvvkVEKW1EYDw9HhTMnP2b/go-lo
```

See 9ac239de3430e0c58b3ee441f5d5bb90fbe6a6f8 for a fixing commit.

**Comment by cryptix** [comment]
Also:

```
➜  go-ipfs git:(go1.6b1) ✗ go test github.com/ipfs/go-ipfs/commands/http
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x30 pc=0x57c2fd]

goroutine 82 [running]:
net/http.(*response).CloseNotify(0xc820302780, 0x0)
	/home/cryptix/gotip/src/net/http/server.go:1543 +0x18d
github.com/ipfs/go-ipfs/commands/http.internalHandler.ServeHTTP.func2(0x8014f1968, 0xc820302780, 0x8014f1928, 0xc82026d940, 0xc820182d70)
	/home/cryptix/go/src/github.com/ipfs/go-ipfs/commands/http/handler.go:155 +0x39
created by github.com/ipfs/go-ipfs/commands/http.internalHandler.ServeHTTP
	/home/cryptix/go/src/github.com/ipfs/go-ipfs/commands/http/handler.go:159 +0x6c1
FAIL	github.com/ipfs/go-ipfs/commands/http	0.043s
```

**Comment by whyrusleeping** [comment]
that CloseNotify panic looks like a boatload of fun...

**Comment by Kubuxu** [comment]
This was resolved in #2315
